ajax/libs/FormChimp/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "FormChimp",
+  "filename": "jquery.formchimp.min.js",
+  "title": "FormChimp",
+  "description": "A customizable MailChimp ajax plugin for jQuery",
+  "version": "1.2.1",
```

Comment:
Let's remove `"version": "1.2.1",` because we add this lib via single package.json
Thank you.